runtime/vm/createramclass.cpp

```diff
@@ -1724,12 +1725,14 @@ loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9
 				/* we will inherit exemption from superclass */
 				isExemptFromValidation = TRUE;
 			}
+			#if JAVA_SPEC_VERSION < 24
```

Comment:
Please don't indent preprocessor directives.

Reply:
Does this apply to Java files as well?

Reply:
In both cases it's just a convention. The vast majority of directives in C/C++ code are not indented: the suggestion was that this be consistent.

Java code is less consistent, but my preference (which isn't always followed) is that directives there should be indented at the same level as the construct they make conditional.